sdks/typescript/test/localParDo.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { registerLocalParDo } from "../src/apache_beam/serialization";
+import * as beam from "../src/apache_beam";
+import { directRunner } from "../src/apache_beam/runners/direct_runner";
+
+export const log = {
+  exportName: 'log',
+  process: function (lines) {
+    console.log(lines);
+    return lines
+  }
+};
+
+registerLocalParDo(log)
```

Comment:
I'm not quite following how we ensure this gets called.

Reply:
in worker_main L64

Reply:
OK, so this is just for DoFns that get called in the main (aka `process.argv[1]`) file? We'd still need serialization capabilities for functions defined elsewhere?